### ISSUE TITLE
weather

### ISSUE THREAD

**Issue by a1wj1** [opendilab/InterFuser#41]
May I ask what the 14 types of weather are like? Can you summarize them, please

**Comment by deepcs233** [comment]
Hi!
Here are all 21 types of weather :https://github.com/opendilab/InterFuser/blob/e4f0314482124bb06a475c3f6fb4bfe3a2701c4d/leaderboard/team_code/base_agent.py#L21
They include different weather conditions (clear, cloudy, rainy, etc.) and different times of the day (noon, night, sunset, etc.)

**Comment by a1wj1** [comment]
Thank you!

**Comment by deepcs233** [comment]
Yes
